histomicstk/cli/NucleiDetection/NucleiDetection.xml

```diff
@@ -218,4 +218,36 @@
       <default>1</default>
     </integer>
   </parameters>
+  <parameters advanced="true">
```

Comment:
As a minor change, can we move these new values to above the Dask block?

Reply:
The rationale is that we have the Dask block last on all of the CLI that have it.